dali/pipeline/data/backend.h

```diff
@@ -78,6 +78,18 @@ inline void MemCopy(void *dst, const void *src, size_t bytes, cudaStream_t strea
   CUDA_CALL(cudaMemcpyAsync(dst, src, bytes, cudaMemcpyDefault, stream));
 }
 
+// This is defined only for backends that map to actual storage devices
+template <typename Backend>
+struct backend_to_storage_dev;
```

Comment:
```suggestion
struct backend_to_storage_device;
```

Reply:
I object to the aforementioned objection, because I find long names objectionable.

Reply:
Done